### PR TITLE
VEGA-2430 Show attorney anomaly for unusable address

### PR DESCRIPTION
**Pull request by nickpyett-djg**
[VEGA-2430](https://opgtransform.atlassian.net/jira/software/projects/VEGA/boards/170?selectedIssue=VEGA-2430) This PR covers showing anomalies in the front-end if an active or replacement Attorney's address is not usable. 

## Checklist

- [x] I have updated the end-to-end tests to work with my changes
- [x] I have added styling for Dark Mode
- [x] I have checked that my UI changes meet accessibility standards

[VEGA-2430]: https://opgtransform.atlassian.net/browse/VEGA-2430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: web/template/layout/mlpa-details-attorneys.gohtml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+              <dt class="govuk-summary-list__key">Date of birth</dt>
+              <dd class="govuk-summary-list__value">
+                {{ if not (eq $attorney.DateOfBirth "") }}
+                  {{ parseAndFormatDate $attorney.DateOfBirth "2006-01-02" "2 January 2006" }}
+                {{ end }}
+              </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+              <dt class="govuk-summary-list__key">Address</dt>
+              {{ $attorneyFieldAnomalies = $attorneyAnomalies.GetAnomaliesForFieldWithStatus "address" "detected" }}
```
Comment (nickpyett-djg) [inline web/template/layout/mlpa-details-attorneys.gohtml:50]: These next few lines have changed from the `mlpa-details` template.

File: web/template/layout/mlpa-details-replacement-attorneys.gohtml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+              <dt class="govuk-summary-list__key">Date of birth</dt>
+              <dd class="govuk-summary-list__value">
+                {{ if not (eq $attorney.DateOfBirth "") }}
+                  {{ parseAndFormatDate $attorney.DateOfBirth "2006-01-02" "2 January 2006" }}
+                {{ end }}
+              </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+              <dt class="govuk-summary-list__key">Address</dt>
+              {{ $attorneyFieldAnomalies = $attorneyAnomalies.GetAnomaliesForFieldWithStatus "address" "detected" }}
```
Comment (nickpyett-djg) [inline web/template/layout/mlpa-details-replacement-attorneys.gohtml:76]: These next few lines have changed from the `mlpa-details` template.

File: cypress/e2e/anomaly.cy.js [hunk 3]
```diff
@@ -99,6 +99,20 @@ describe("View and edit anomalies for a digital LPA", () => {
...
+            status: "detected",
+            fieldName: "address",
+            ruleType: "address-not-usable",
+            fieldOwnerUid: "attorney-1-uid",
+          },
+          {
+            id: 131,
+            status: "detected",
+            fieldName: "address",
+            ruleType: "address-not-usable",
```
Comment (MishNajam) [inline cypress/e2e/anomaly.cy.js:113]: this doesn't impact the test but ideally the ruleType would be updated to invalidAddress...could you also update anomaly 129 😓

Comment (nickpyett-djg) [reply cypress/e2e/anomaly.cy.js]: Good find! This has been fixed, cheers.

**Review by MishNajam — APPROVED** [review]

**Pull request merged**